app/mailers/notifications.rb

```diff
@@ -21,7 +21,7 @@ def import_awaiting_approval(import)
     @import = import
     mail(
       :to => moderators, 
-      :subject => "A Safecast import is awaiting approval")
+      :subject => "A Safecast import is awaiting approval - " << import.filename)
```

Comment:
This is okay, but using string interpolation is more ruby-ish. For example, `"A Safecast ... approval - #{import.filename}"`